### PR TITLE
Feature/gh actions

### PR DESCRIPTION
**Pull request by Baklap4**
Adding GH-Actions to auto-deploy instead of Semaphore

### CODE REVIEW

**Comment by Baklap4** [comment]
@ooshlablu please look at this :)

File: .github/workflows/build_and_release.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+name: Build and Release Performous Website
+
+# Controls when the workflow will run
+on:
+  # Run on a schedule to get weekly updates for the Linux containers
+  # and keep the cache fresh for Windows builds. Runs Sundays at midnight.
+  schedule:
```
Comment (ooshlablu) [inline .github/workflows/build_and_release.yml:7]: I'm not sure that the schedule is useful for the website. I don't think there will be any changes that sneak in that don't get deployed when master is run :-)

File: .github/workflows/build_and_release.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      - name: Setup Pages
+        uses: actions/configure-pages@v3
+
+      - name: Download artifacts
+        uses: actions/download-artifact@v2
+        with:
+           name: web-output
+           path: './web-output'
+
+      - name: Upload artifact
```
Comment (ooshlablu) [inline .github/workflows/build_and_release.yml:68]: Should this `upload artifact` step be part of the deployment?

**Review by ooshlablu — CHANGES_REQUESTED** [review]
I have a couple of questions, maybe they are no-ops tho.

File: .github/workflows/build_and_release.yml (hunk 2 above)
Comment (Baklap4) [reply .github/workflows/build_and_release.yml:68]: Yeah i've added this so the pr's get an artifact (we can even add comment_pr thingy to make them publicly available but i didn't do that yet). This way you can download it, and try it locally before merging 👍

File: .github/workflows/build_and_release.yml (hunk 1 above)
Comment (Baklap4) [reply .github/workflows/build_and_release.yml:7]: True that: fixed

**Review by ooshlablu — APPROVED** [review]
LGTM!

**Pull request merged**